brigade/core/task.py

```diff
@@ -93,11 +101,12 @@ def grouped_tasks(task):
         if "severity_level" not in kwargs:
             kwargs["severity_level"] = self.severity_level
         r = Task(task, **kwargs).start(self.host, self.brigade)
+        self.results.append(r[0] if len(r) == 1 else r)
+
         if r.failed:
             # Without this we will keep running the grouped task
-            raise r.exception
+            raise BrigadeSubTaskError((r.name))
```

Comment:
Given my [previous](https://github.com/brigade-automation/brigade/pull/131/files#r185722111) comment I think this should be:

```
raise BrigadeSubTaskError(task=task, error=r)
```

That will also mean we have to change:

```
r = Task(task, **kwargs).start(self.host, self.brigade)
```

to:

```
task = Task(task, **kwargs)
r = task.start(self.host, self.brigade)
```